nevergrad/functions/games/battleship.py

```diff
@@ -0,0 +1,337 @@
+from __future__ import absolute_import, division, print_function, unicode_literals
```

Comment:
this is python2, you should remove it
also, please add nevergrad's header

Reply:
done!